daemon/api_debug.go

```diff
@@ -36,10 +36,10 @@ import (
 )
 
 var debugCmd = &Command{
-	Path:   "/v2/debug",
-	UserOK: true,
-	GET:    getDebug,
-	POST:   postDebug,
+	Path:     "/v2/debug",
+	RootOnly: true,
```

Comment:
I think this should remain UserOK. For pprof there's a separate debug command with PathPrefix set to /v2/debug/pprof added right before the actual debugCmd. I think you can try and register a new api.Command with Path set to /v2/debug/stacktrace and RootOnly, but add it right before debugCmd.

Reply:
Done